pkg/api/resource/suffix.go

```diff
@@ -83,6 +83,8 @@ func newSuffixer() suffixer {
 	// a suffix for 2^0.
 	sh.decSuffixes.addSuffix("", bePair{2, 0})
 
+	sh.decSuffixes.addSuffix("n", bePair{10, -9})
+	sh.decSuffixes.addSuffix("μ", bePair{10, -6})
```

Comment:
Copy-paste, of course. =P

More seriously. @timstclair there's some carefully thought-out logic in the code of this object around accessing functions. Adding suffixes here is not sufficient. Please add to the currently-hopefully-exhaustive test cases.

...You will also have to figure out what to do about making MilliValue() analogs.